shell/browser/api/electron_api_web_contents_mac.mm

```diff
@@ -52,9 +53,20 @@ - (void)redispatchKeyEvent:(NSEvent*)event;
     return false;
 
   // Send the event to the menu before sending it to the window
-  if (event.os_event.type == NSEventTypeKeyDown &&
-      [[NSApp mainMenu] performKeyEquivalent:event.os_event])
-    return true;
+  if (event.os_event.type == NSEventTypeKeyDown) {
+    // If the keyboard event is a system shortcut, it's already sent to the
+    // NSMenu instance in
+    // content/app_shim_remote_cocoa/render_widget_host_view_cocoa.mm via
+    // keyEvent:(NSEvent*)theEvent wasKeyEquivalent:(BOOL)equiv. If we let the
+    // NSMenu handle it here as well, it'll be sent twice with unexpected side
+    // effects.
+    auto modifier_mask = ui::cocoa::ModifierMaskForKeyEvent(event.os_event);
+    if ((modifier_mask & NSEventModifierFlagFunction) != 0)
```

Comment:
I think we should also do a test of `[[NSApp mainMenu] cr_menuItemForKeyEquivalentEvent:theEvent] != nil` like the original Chromium change did, otherwise other valid Fn key events won't be sent to renderer.

Reply:
@zcbenz done!